src/docs/uxd/toolbarsFilters/DOC.md

```diff
@@ -1,5 +1,102 @@
 # Toolbars & Filters
 
-Content TBD
+![alt text](https://github.com/RedHatInsights/insights-frontend-storybook/blob/master/src/docs/uxd/img/toolbarLayout.png?raw=true "toolbarLayout")
 
-[Google Doc](https://docs.google.com/document/d/1Uq1cmLxrUBhhDZrUe7KIdesLKORT2IF5M13XqCJ6yYY/edit?usp=sharing)
+1. **Filtering**: Filtering should be on the left-most side of the toolbar. All lists should have some filtering. (See filtering section for more details)
+2. **Action buttons**: Arrange action buttons to the right of the filtering. There can be 1 primary action (an action is not required), 1 secondary action, and any overflow goes into the kebab.
+3. **Kebab & Export**: Export should always go into the kebab, even if it is the only action on the table or list.
+4. **Pagination**: Use pagination on top and bottom of table. [PF4 pagination](https://v2.patternfly.org/documentation/core/components/pagination) will responsively react to show only the results count when in smaller spaces. This is the only results count shown. (See filtering section for more details about how the results count works with filtering)
```

Comment:
Need to update this link to be to the new patternfly website.

If you see a reference to any pattern/doc that is not the base patternfly.org (any v2s, or v4s) then search on the patternfly website for the updated link. A lot of this stuff was written before the new pf website was up so there are some dead links. whoops.